$FreeBSD$

Hack configure to eat autogenerated qt options and to fix a bashism (type -p).

--- configure.orig	Wed Dec 10 03:10:03 2003
+++ configure	Wed Jan  7 16:30:11 2004
@@ -81,6 +81,8 @@
 			shift
 			;;
 		--help) show_usage; exit ;;
+		--with-*)
+			shift ;;
 		*) show_usage; exit ;;
 	esac
 done
@@ -136,7 +138,7 @@
 		echo Warning: qmake not in \$QTDIR/bin/qmake
 		echo trying to find it in \$PATH
 	fi
-	qm=`type -p qmake`
+	qm=`which qmake`
 	if [ -x "$qm" ]; then
 		if [ "$QC_DEBUG" = "Y" ]; then
 			echo qmake found in $qm
